rpm_spec/subpackages/manageiq-ansible-venv

```diff
@@ -0,0 +1,16 @@
+%package ansible-venv
+Summary: %{product_summary} Ansible Runner Virtual Environment
+
+%global __requires_exclude ^\/var\/lib\/manageiq\/venv\/bin\/python
```

Comment:
The venv bin files will have `#! /var/lib/manageiq/venv/bin/python` as the shebang and that is expected.  Thus we need this exclude otherwise rpmbuild will assume that shebang is a Require and that creates a circular reference (since this file is shipped as part of the rpm itself).